modules/lambdas/main.tf

```diff
@@ -127,7 +127,7 @@ resource "aws_lambda_permission" "delete_old_reconcile_jobs_allow_cloudwatch_eve
 resource "aws_lambda_function" "get_current_archive_list" {
   ## REQUIRED
   function_name = "${var.prefix}_get_current_archive_list"
-  role          = var.restore_object_role_arn
+  role          = var.gql_tasks_role_arn
```

Comment:
All other lambdas are using the `restore_object_role_arn` IAM role. Can we just update the `restore_object_role_arn` by adding the necessary permissions you added for `gql_tasks_role_arn' instead of cahnging roles? 

That way we can be consistent with using just one IAM role for all lambas.

Reply:
We could possibly, but then ECS would have to use that restore_object_role_arn as well for graphql to be able to do it's S3 Imports which I'm not sure how that would affect users using multiple deployments on one database with those changes we made for that

Reply:
ok sounds good